packages/server/src/service/contact/loader.ts

```diff
@@ -29,16 +29,24 @@ export const loadStates = async (): Promise<RawContactRecord> => {
   return ret
 }
 
-export const loadMichigan = async (
-  data: Record<string, RawContact>
+export const loadStateCounties = async (
+  data: Record<string, RawContact>,
+  state: ImplementedState,
 ): Promise<Record<string, RawContact & { key: string }>> => {
   return Object.fromEntries(
     Object.entries(data)
       .map(([key, rec]) => {
-        if (!rec.county) throw Error('Encountered Michigan record without count field during loading')
+        // Since the ArcGIS for wisconsin has some data that the election
```

Comment:
Are their records without counties in Wisconsin?  If so -- let's only except those (write a list of them here).  I'm worried about exempting all of Wisconsin.